### PR TITLE
Personal data is also information relating to a person

### PR DESCRIPTION
**Pull request by julianladisch**
Personal data is also information _**relating to**_ a person.

The Data Protection Authority in Ireland (Ireland is a EU member state) explains personal data:
https://www.dataprotection.ie/en/dpc-guidance/what-is-personal-data

### CODE REVIEW

**Comment by mdgatid** [comment]
Sorry Julian - I'm not following what you are changing here?

**Comment by julianladisch** [comment]
When you look at "Files changed" https://github.com/folio-org/personal-data-disclosure/pull/3/files the red line show the content I would like to get removed and replaced, and the green lines show the new content.
The most important change is that data about a person also is considered "personal data".
Sorry for using British English. Translating the title "Personal data is also information relating to a person" into American English yields "Personal data also includes information that relates to an individual.".

File: PERSONAL_DATA_DISCLOSURE.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 ## Overview
 The purpose of this form is to disclose the types of personal data stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
 
-It's important to note that personal data is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender).
+GDPR defines: ‘personal data’ means any information relating to an identified or identifiable natural person.
+
+Personal data basically means any information about a living person, where that person either is identified or could be identified.
+
+It's important to note that personal data is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any information about such a person (has loaned 5 items, 2 of which are overdue).
```
Comment (craigmcnally) [inline PERSONAL_DATA_DISCLOSURE.md]:
```suggestion
It's important to note that personal data is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any information about such a person (e.g. has loaned 5 items, 2 of which are overdue).
```

**Comment by craigmcnally** [comment]
Fine by me.  Please increment the document version to 1.1 at the bottom of the file.

**Review by craigmcnally — APPROVED** [review]

**Comment by julianladisch** [comment]
Someone else need to merge this pull request as I don't have write access to this repository.

**Comment by jwfuller** [comment]
@julianladisch The Privacy SIG has updated the form and tried to include this suggestion. Does https://github.com/folio-org/personal-data-disclosure/blob/master/PERSONAL_DATA_DISCLOSURE.md?plain=1#L5= cover the changes in this PR?

**Comment by julianladisch** [comment]
Yes.

**Pull request closed without merging**